deezer_downloader/deezer.py

```diff
@@ -12,6 +13,8 @@
 import html.parser
 import requests
 from binascii import a2b_hex, b2a_hex
+from mutagen.flac import FLAC, Picture
+from mutagen.id3 import ID3, APIC
```

Comment:
also both not used right now, but I would like to replace the current ID3 tag implementation for mp3s by using  `mutagen.id3.ID3` library